maitake/README.md

```diff
@@ -132,10 +132,13 @@ The following features are available (this list is incomplete; you can help by [
 
 [expanding it]: https://github.com/hawkw/mycelium/edit/main/maitake/README.md
 
-| Feature | Default | Explanation |
-| :---    | :---    | :---        |
-| `alloc` | `true`  | Enables [`liballoc`] dependency |
+| Feature        | Default | Explanation |
+| :---           | :---    | :---        |
+| `alloc`        | `true`  | Enables [`liballoc`] dependency |
 | `no-cache-pad` | `false` | Inhibits cache padding for the [`CachePadded`] struct. When this feature is NOT enabled, the size will be determined based on target platform. |
+| `tracing-01`   | `false`  | Enables support for v0.1.x of [`tracing`] (the current release version). Requires `liballoc`.|
```

Comment:
Wait, does tracing 0.1 still require `liballoc`? I thought we had squished that away. I was able to build a no-std lib for thumbv7em-none-eabihf with tracing v0.1.35.

Reply:
Tracing 0.1 will always require liballoc, it supports no-std + alloc. Getting rid of alloc in 0.1 isn't possible as it would require feature-flagging APIs that are currently available on `default-features = false`.